File: 15_FullStack/27.05.2024/ClassExercise/frontend/src/Components/Model/Donation.ts
export class Donation{
    public id:number;
    public name:string;
    public donationAmount:number;
    public blessing:string;

    constructor(id:number,name:string,donationAmount:number,blessing:string){
        this.id=id;
        this.name=name;
        this.donationAmount=donationAmount;
        this.blessing=blessing;
    }
}

File: 15_FullStack/27.05.2024/ClassExercise/donation/src/main/java/com/johnbryce/donation/beans/Donation.java
package com.johnbryce.donation.beans;

import jakarta.annotation.Nullable;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
@Builder
public class Donation {
    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    private int id;
    private String name;
    @Column(nullable = false)
    private long donationAmount;
    private String blessing;
}